snapm/manager/plugins/stratis.py

```
@@ -505,8 +532,16 @@ def discover_snapshots(self):
                 continue
 
-            pool_name = path_to_name[filesystem.Pool()]
-            filesystem_name = str(filesystem.Name())
+            try:
+                pool_name = path_to_name[filesystem.Pool()]
+            except DbusClientMissingPropertyError:
+                self._log_warn("Could not access filesystem.Pool() property")
+                continue
+            try:
+                filesystem_name = str(filesystem.Name())
+            except DbusClientMissingPropertyError:
```

Ruff (0.15.9)

[error] Undefined name `DbusClientMissingPropertyError`

(F821)